lib/sda/tree.go

```diff
@@ -258,7 +259,7 @@ func (tm TreeMarshal) MakeTree(il *EntityList) (*Tree, error) {
 
 // MakeTreeFromList creates a sub-tree given an EntityList
 func (tm *TreeMarshal) MakeTreeFromList(parent *TreeNode, il *EntityList) *TreeNode {
-	idx, ent := il.Search(tm.EntityId)
+	idx, ent := il.SearchEntity(tm.EntityId)
```

Comment:
@ineiti  said :
>  that it's possible that one EntityIdx doesn't have to be unique in a tree!

For my part, the function name I would keep `Search()`, as you call this function from an `EntityList`, so it's twice the same information. It's like method name for packages, if you have `schnorr.SignSchorr`, I find it quite redundant...